lib/game.rb

```diff
@@ -0,0 +1,26 @@
+class Game
+  attr_reader :stat_hash
+
+  def initialize()
+    @stat_hash = {}
```

Comment:
Where does this call from?

Reply:
line 11 calls stat_hash and creates a key out of the first element in the first line of the data file